src/Utils/Helpers.ts

```diff
@@ -1,4 +1,5 @@
 import { ok } from 'assert';
+import * as path from 'path';
```

Comment:
Don't use wildcard import 
```suggestion
import { parse } from 'path';
```